backend/courses/models.py

```diff
@@ -150,6 +152,17 @@ def __str__(self):
     class Meta:
         unique_together = ['institution', 'external_id']
 
+class Team(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid4, editable=False)
+    name = models.CharField(max_length=255, blank=False, null=False)
```

Comment:
Do we want to allow students to name their teams, or is the expected behavior for front end or the api to generate team names?

Reply:
Maybe either? Can add it as a flag in the course Team config model. Either way, if we do want auto generated names they'll be decided by endpoint logic
We can discuss with Eyolfson with what he thinks is best